invenio_sip2/api_views.py

```diff
@@ -0,0 +1,129 @@
+# -*- coding: utf-8 -*-
+#
+# INVENIO-SIP2
+# Copyright (C) 2020 UCLouvain
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Blueprint for Invenio-SIP2."""
+
+from __future__ import absolute_import, print_function
+
+from flask import Blueprint, jsonify
+
+from .records.record import Client, Server
+
+api_blueprint = Blueprint(
+    'api_sip2',
+    __name__,
+    url_prefix='/monitoring/sip2'
+)
+
+
+@api_blueprint.route('/status')
+def status():
+    """Displa status for all SIP2 server."""
+    return jsonify(Monitoring.status())
+
+
+@api_blueprint.route('/servers')
+def get_servers():
+    """Display all running SIP2 servers."""
+    try:
+        servers = Server.get_all_records()
+        print('servers:', servers)
+        return jsonify({'servers': servers})
```

Comment:
`return jsonify({'servers': Server.get_all_records()})` ? Only if previous line is debug ;-)

Reply:
done